src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs

```diff
@@ -306,15 +306,15 @@
 (react/defc NotebooksContainer
   {:refresh
    (fn [{:keys [this]}]
-     (this :-get-clusters-list))
+     (this :-get-clusters-list-if-whitelisted))
    :render
    (fn [{:keys [props state this]}]
      (let [{:keys [server-response]} @state
```

Comment:
Incidentally, you might `:keys` out `show-create-dialog?` here, while you're at it.